articles/tech-resources/a-b-testing-sparkpost.md

```diff
@@ -51,9 +184,9 @@ A/B testing of email is not immediate - it takes some amount of time for opens a
   * Offer language/different offers
   * Header image 
 
-**Planned Enhancements and known limitations**
+**Planned enhancements and known limitations**
 
- *  The current "Learning Mode" does not automatically pick a winner. It provides real-time results via webhooks to allow users to analyze the results and make a determination which template is the best based on open or click rates. It is then a separate call to the A/B Testing API to change the default template to be the winning template. The next enhancement is the introduction of a statistical model in Bayesian Mode that will pick the winner automatically for you. This is coming soon!
+ *  "Learning Mode" does not automatically pick a winner. It provides real-time results via webhooks to allow users to analyze the results and make a determination which template is the best based on open or click rates. It is then a separate call to the A/B Testing API to change the default template to be the winning template.
```

Comment:
Let's just take "Learning Mode" out of this section.  It's not a limitation now that we have Bayesian.
Also what happened to the listing of "only single recipient REST transmissions, only stored templates . . ."